src/libaktualizr/storage/sql_utils.h

```diff
@@ -54,7 +54,8 @@ class SQLiteStatement {
     if (b == nullptr) {
       return boost::none;
     }
-    return std::string(b);
+    size_t length = static_cast<size_t>(sqlite3_column_bytes(stmt_.get(), iCol));
```

Comment:
This is a great catch, thanks! However, CI is complaining about static checks:

```
src/libaktualizr/storage/sql_utils.h:57:5: error: use auto when initializing with a cast to avoid duplicating the type name [modernize-use-auto,-warnings-as-errors]
    size_t length = static_cast<size_t>(sqlite3_column_bytes(stmt_.get(), iCol));
    ^~~~~~
    auto
```